src/main/resources/sql/parm.sql

```diff
@@ -0,0 +1,57 @@
+select parm.parm_cd, 
+	parm.parm_nm, 
+	parm.parm_rmk_tx,
+	parm.parm_unt_tx,
+	parm.parm_seq_nu,
+	parm.parm_seq_grp_cd,
+	parm.parm_ds,
+	parm.parm_medium_tx,
+	parm.parm_frac_tx,
+	parm.parm_temp_tx,
+	parm.parm_stat_tx,
+	parm.parm_tm_tx,
+	parm.parm_wt_tx,
+	parm.parm_size_tx,
+	parm.parm_entry_fg,
+	parm.parm_public_fg,
+	parm.parm_neg_fg,
+	parm.parm_zero_fg,
+	parm.parm_null_fg,
+	parm.parm_ts_fg,
+	parm.parm_init_dt,
+	parm.parm_init_nm,
+	parm.parm_rev_dt,
+	parm.parm_rev_nm,
+	parm_seq_grp_cd.parm_seq_grp_nm, 
+	pa.wqpcrosswalk,
+	pa.srsname
+from parm
+left join parm_seq_grp_cd
+         on regexp_replace(parm.parm_seq_grp_cd, '(^[[:space:]]*|[[:space:]]*$)') = regexp_replace(parm_seq_grp_cd.parm_seq_grp_cd, '(^[[:space:]]*|[[:space:]]*$)')
+join
```

Comment:
This is the join causing Laura's issue... (wrong counts on USGS activity pulls) No suggestions at this time.